src/test/java/com/mathworks/ci/MatlabInstallationTest.java

```diff
@@ -85,4 +122,90 @@ public void verifyInstallationInPipeline() throws Exception {
         jenkins.assertBuildStatusSuccess(build);
         jenkins.assertLogContains("versioninfo", build);
     }
+    /*
```

Comment:
at line 82 in this file we are using  matlabInstDescriptor.get installations(); and in MatlabInstallation class at line 76 we also have similar static method which returns the same we do you think these methods are be redundant in usage ? 
for eg : if we already have a static method we could use that across the code whenevr we need array of Installations
matlabInstDescriptor.getInstallations()  --> MatlabInstallations.getAll() just to keep the implementation consistent

Reply:
Done.